diff --git a/applet/networkmanager.cpp b/applet/networkmanager.cpp
index ab4e251..e2c15ee 100644
--- a/applet/networkmanager.cpp
+++ b/applet/networkmanager.cpp
@@ -30,7 +30,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #include <QGraphicsPixmapItem>
 #include <QTimeLine>
 
-#include <QGraphicsBlurEffect>
+//#include <QGraphicsBlurEffect>
 
 #include <KIcon>
 #include <KIconLoader>
@@ -79,7 +79,7 @@ NetworkManagerApplet::NetworkManagerApplet(QObject * parent, const QVariantList
     setHasConfigurationInterface(false);
     setPopupIcon(QIcon());
     setPassivePopup(true); // FIXME: disable, only true for testing ...
-    m_overlayTimeline.setEasingCurve(QEasingCurve::OutExpo);
+    //m_overlayTimeline.setEasingCurve(QEasingCurve::OutExpo);
     m_currentState = 0;
     connect(&m_overlayTimeline, SIGNAL(valueChanged(qreal)), this, SLOT(repaint()));
 
